specs/fjord/exec-engine.md

```diff
@@ -22,7 +22,8 @@ The L1 cost is computed as:
 
 ```pseudocode
 l1FeeScaled = l1BaseFeeScalar*l1BaseFee*16 + l1BlobFeeScalar*l1BlobBaseFee
-l1CostSigned = (intercept + fastlzCoef*fastlzSize + txSizeCoef*txSize) * l1FeeScaled / 1e12
+estimatedLen = intercept + fastlzCoef*fastlzSize 
+l1CostSigned = max(71, estimatedLen) * l1FeeScaled / 1e12
```

Comment:
I'd prefer to make the `71` a named parameter in the specs, like `fastlzMinCost`. It might also still change.

I think we can also now rename `l1CostSigned` to `l1Cost` because we now already guarantee in its definition that it will be > 0.